### ISSUE TITLE
I'd like to ... add traits information to member profiles 

### ISSUE THREAD

**Issue by jhpoelen** [open-traits-network/open-traits-network.github.io#197]
Hi! 

In addition to associating OTN members to their taxa of interest, we'd like to associate OTN member to the traits they are interested in . 

Suggest to follow similar approach as taken in #14 .

**Comment by jhpoelen** [comment]
@seltmann can you please suggest some bee traits that you are studying? (e.g., nose size?*) 

@Daniel-Mietchen would like to use bee traits as a way to get started with trait - member - publication relations with a particular interest in invasion biology (if possible). 

* disclaimer: I am not a bee expert :smile:

**Comment by Daniel-Mietchen** [comment]
@bmaitner these links give certificate errors, but I had a first try at curating the Scholia topic profile for seed mass (focusing on papers that had the term in their title): https://scholia.toolforge.org/topic/Q113141694 . Filtered for OTN members, this yielded only two: https://scholia.toolforge.org/organization/Q112326635/topic/Q113141694 .

Similar results for 
- seed dispersal (4): https://scholia.toolforge.org/organization/Q112326635/topic/Q943313
- seed size (3): https://scholia.toolforge.org/organization/Q112326635/topic/Q113142361 
- seed dormancy (2): https://scholia.toolforge.org/organization/Q112326635/topic/Q2397491

I also looked into traits more generally, and while their coverage and modelling in Wikidata is currently both patchy and inconsistent, some OTN-related patterns are [beginning to emerge](https://w.wiki/5UYb):
![Screenshot from 2022-07-20 03-10-03](https://user-images.githubusercontent.com/465923/179874242-5910f9f7-1776-4f33-85be-c473495053e0.png)

**Comment by bmaitner** [comment]
Awesome work!  I wonder if part of the reason you didn't find much using the traits I suggested is that they aren't mentioned in the title when multiple traits are used.  Alternatively, more general terms might be used in lieu of more precise terms.  For example,  many papers may refer to the "leaf economic spectrum" which can encompass a wide suite of traits, or might say "leaf size" when they mean "leaf area".  I wonder if we might have to bring in trait ontologies to help sort this out?

**Comment by Daniel-Mietchen** [comment]
@bmaitner Yes, we should probably have a good look at integrating with relevant ontologies, and both TPO and PO are on my radar. I did not actually try the other terms last time, as I am avoiding interactions with websites that have certificate problems.

In any case, I have now also seeded some Scholia profiles for
- specific leaf area: https://scholia.toolforge.org/topic/Q7574996
- leaf mass per area: https://scholia.toolforge.org/topic/Q113187106 
- leaf dry matter: https://scholia.toolforge.org/topic/Q113023152

**Comment by Daniel-Mietchen** [comment]
Here's one for plant growth form as well: https://scholia.toolforge.org/topic/Q113187687

**Comment by Daniel-Mietchen** [comment]
Here we go, finally, for an initial profile of plant height: https://scholia.toolforge.org/topic/Q113205681 .
